core/store/migrate/migrations/0065_ensure_chain_specific_gas_limit.sql

```diff
@@ -1,9 +1,10 @@
 -- +goose Up
 -- +goose StatementBegin
 
--- NOT VALID is here to grandfather in old attempts which inadvertently saved
--- with a 0 value, but to enforce correctly writing data for future attempts
-ALTER TABLE eth_tx_attempts ADD CONSTRAINT chk_chain_specific_gas_limit_not_zero CHECK (chain_specific_gas_limit > 0) NOT VALID;
+-- Grandfather in old attempts which inadvertently saved with a 0 value, and
+-- enforce correctly writing data for future attempts
+UPDATE eth_tx_attempts SET chain_specific_gas_limit=1 WHERE chain_specific_gas_limit=0;
```

Comment:
There's no way for goose down to restore these, is there?

Reply:
No and it wouldn't matter anyway, its unused in older versions